docs/using-docker.rst

```diff
@@ -10,20 +10,18 @@ Install Docker onto the machine you plan to run Teleirc from.
 Which image do I choose?
 ************************
 
-Node Slim, Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
+Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
 
 +-----------------------------------------------------------------------------+-----------------------+---------+
 | Image                                                                       | File                  | Size    |
 +=============================================================================+=======================+=========+
 | `Node Alpine Linux <https://hub.docker.com/r/_/node/>`_ (``node:6-alpine``) | ``Dockerfile.alpine`` | 66.9 MB |
 +-----------------------------------------------------------------------------+-----------------------+---------+
-| `Node Slim <https://hub.docker.com/r/_/node/>`_  (``node:6-slim``)          | ``Dockerfile.slim``   | 256 MB  |
-+-----------------------------------------------------------------------------+-----------------------+---------+
 | `Fedora latest <https://hub.docker.com/r/_/fedora/>`_                       | ``Dockerfile.fedora`` | 569 MB  |
 +-----------------------------------------------------------------------------+-----------------------+---------+
 
 This guide uses ``alpine``.
-If you use another image, replace ``alpine`` with ``slim`` or ``fedora``.
+If you wish to use ``fedora``, replace ``alpine`` with ``fedora``.
 
 You will see errors during ``npm install``.
```

Comment:
Does `npm install` still apply if we are using `yarn`?

Reply:
``yarn` is `npm install`.